packages/design-system-react/src/components/button-link/ButtonLink.types.ts

```diff
@@ -0,0 +1,47 @@
+import type { ButtonBaseProps } from '../button-base';
+import { ButtonBaseSize } from '../button-base';
+
+export enum ButtonLinkSize {
+  Sm = ButtonBaseSize.Sm,
+  Md = ButtonBaseSize.Md,
+  Lg = ButtonBaseSize.Lg,
+  /**
+   * Inherits size from parent, removes height/padding, displays inline
```

Comment:
probably needs to update the description here

Reply:
It will still inherit the font size but I think we could make it clearer by changing it to
```suggestion
   * Inherits font size from parent, removes height/padding, displays inline
```
What do you think?

Reply:
Updated to include `font size`